themes/openy_themes/openy_carnation/templates/paragraph/paragraph--4c.html.twig

```diff
@@ -41,14 +41,15 @@
     'paragraph',
     'block-description--wrapper',
     'block-description--4',
-    'row'
```

Comment:
Shouldn't there be a row inside a container, as in paragraph--2c.html.twig, /paragraph--3c.html.twig ?